SampleReports/report-inverted.md

```diff
@@ -0,0 +1,56 @@
+# Retire Vulnerability Report
```

Comment:
I think I do have that info here.

The first item is Microsoft.AspNetCore.SpaServices, which says 'from dependencies' of 'Microsoft.AspNetCore.SpaServices' because it is the root dependency.

Further down, we ca see System.IO.Compression.ZipFile has an issue, and that is included from Microsoft.AspNetCore.SpaServices.

That's how I read the data model -- did I miss something? The goal was to trim it down to 'problem library' and 'source of problem inclusion', since one of the first comments I got from one of the devs I work with is 'that is a great big wall of text' regarding the 'markdown' version of the report :)

Reply:
Ah, I see, I just misread it then. Thanks!